back/src/settings/route.ts

```diff
@@ -3,18 +3,25 @@ import passport from 'passport';
 import {Express } from 'express'
 import * as controller from '../app/controllers/index'
 import {Request, Response} from "express";
-import { check} from 'express-validator'
 
 export default function setRoute(connection: Connection, app: Express) {
 
 
 	app.get('/test', (req: Request, res: Response) => {return res.send({coucou: 'salut'});})
 
 
-	app.route('/user').get([check('coucou').isLength({ min: 5 })], controller.user.test);
```

Comment:
route function of express that I told you

Reply:
[check('coucou').isLength({ min: 5 })], ca me parai bizarre de faire ca ici

Reply:
@jeromefrench  C'est des vieux tests, je les laisse pour l'instant.

Reply:
app.route('/user').get('/create', controller.user.create);
On est bien d'accord c'est ca que tu souhaite ?
Car j'arrive pas a la faire compiler cette ligne.
Et la doc est pas hyper claire, donc si tu as une idée hesite pas

Reply:
ok je te montrerais

Reply:
t'as raison @jeromefrench. c'etait plus pour un POC que je l'avais mis là ce truc. mais ça n'a rien a faire là en effet